### DISCUSSION TITLE
LTBEEF Not patched?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1230] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I tested it on a personal computer (Windows 10, lenovo thinkpad) with the CompactCow UI, it still switches extensions off on https://chrome.google.com/webstorex, but idk why it doesn't work on chromebook.

**Comment by mauzyice2** [comment]
it's because it's a school Chromebook dumbass

**Comment by SlitheringLizard1** [comment]
chrome os

that's why
